test/archethic/contracts/interpreter/version1/action_interpreter_test.exs

```diff
@@ -294,25 +294,25 @@ defmodule Archethic.Contracts.Interpreter.Version1.ActionInterpreterTest do
                |> ActionInterpreter.parse()
     end
 
-    # test "should be able to use loop" do
-    #   code = ~S"""
-    #   actions triggered_by: transaction do
-    #     result = 0
-
-    #     for var of [1,2,3] do
-    #         result = var
-    #     end
-
-    #     Contract.set_content result
-    #   end
-    #   """
-
-    #   assert {:ok, :transaction, _} =
-    #            code
-    #            |> Interpreter.sanitize_code()
-    #            |> elem(1)
-    #            |> ActionInterpreter.parse()
-    # end
+    test "should be able to use loop" do
+      code = ~S"""
+      actions triggered_by: transaction do
+        result = 0
+
+        for i: [1,2,3] do
```

Comment:
Shouldn't it be easier to understand or read with: `for i in [1, 2, 3]` ?

Reply:
It's actually pretty easy to change, I started with the  `in` and changed for this syntax later. Just tell me and I'll update it

Reply:
Yes indeed in JS it's more like for ..of but I prefer the syntax and meaning of for in